srcpkgs/umoci/patches/which.patch

```diff
@@ -0,0 +1,23 @@
+From 36ca035cc8cb96b9c0febebcde4d5f4fc82a6022 Mon Sep 17 00:00:00 2001
+From: Cameron Nemo <CameronNemo@users.noreply.github.com>
+Date: Sun, 29 Aug 2021 22:20:27 -0700
+Subject: [PATCH] fix(Makefile): avoid usage of which
+
+which is not available in some build environments
+---
+ Makefile | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/Makefile b/Makefile
+index d76e37b4..3df909c2 100644
+--- a/Makefile
++++ b/Makefile
+@@ -14,7 +14,7 @@
+ # limitations under the License.
+ 
+ # Use bash, so that we can do process substitution.
+-SHELL := $(shell which bash)
++SHELL := $(shell command -v bash)
```

Comment:
We can add `which` to hostmakedepends. Do you think upstream is going to accept the patch (https://github.com/opencontainers/umoci/pull/376)?

Reply:
I think they probably will.